packages/apps/patterns/react-ui/src/components/IdentityList/SpaceMemberListContainer.tsx

```diff
@@ -20,8 +20,12 @@ export const SpaceMemberListContainer = ({ spaceKey, includeSelf, onSelect }: Sp
   const members = useMemo(
     () =>
       includeSelf
-        ? allUnsortedMembers.sort((a) => (a.identity.identityKey.equals(client.halo.identity!.identityKey) ? -1 : 1))
-        : allUnsortedMembers.filter((member) => !member.identity.identityKey.equals(client.halo.identity!.identityKey)),
+        ? allUnsortedMembers.sort((a) =>
+            a.identity.identityKey.equals(client.halo.identity.get()!.identityKey) ? -1 : 1
```

Comment:
Identity isn't immutable though, because it contains your profile which can change. The event also fires when your identity is created or destroyed on a device as well.

There's going to be other, non-collection examples as well (e.g., status or network or status of a space)